docs/config.json

```diff
@@ -484,6 +474,27 @@
               "slug": "/management/guides/ssh-key-extensions/"
             }
           ]
+        },
+        {
+          "title": "Export Teleport Audit Events",
```

Comment:
This title seems to not line up with the other titles like Admin, Operations.  Perhaps Audit Events?  The Teleport part is also not required

Reply:
This was my attempt to nudge this section toward #17556, but you're right that it doesn't really fit. I'll change it.